app/res/xml/preferences_developer.xml

```diff
@@ -130,5 +130,10 @@
         android:entryValues="@array/pref_enabled_vals"
         android:key="cc-enable-bulk-performance"
         android:title="Enable cutting-edge performance improvements"/>
-
+    <ListPreference
```

Comment:
Hm. I'm trying to remember whether we snuck in this being on by default for LTS only in 2.27, which is why it didn't get ported over.

As ported currently this implementation looks correct.